boot/libs.ml

```diff
@@ -1,6 +1,6 @@
 let executables = [ "main" ]
 
-let external_libraries = [ "unix"; "threads.posix" ]
+let external_libraries = [ "unix"; "threads"; "threads.posix" ]
```

Comment:
Did you commit this accidentally? I think this change needs to be reverted and everything should work in CI.

Reply:
I don't know how this got here, will revert

Reply:
Something similar happened to me also. When you do make bootstrap or make dune.exe.

Reply:
now reverted